bluemira/equilibria/flux_surfaces.py

```diff
@@ -850,12 +855,18 @@ def calculate_connection_length_fs(
     passing through Coils, but really they should be intercepted beforehand!
     """
     if first_wall is None:
-        x1, x2 = eq.grid.x_min, eq.grid.x_max
-        z1, z2 = eq.grid.z_min, eq.grid.z_max
+        x1, x2 = eq.grid.x_min + 0.01, eq.grid.x_max - 0.01
+        z1, z2 = eq.grid.z_min + 0.01, eq.grid.z_max - 0.01
```

Comment:
Why add 0.01?

Reply:
If we don't come slightly in from the grid edge then the flux line does not intersect.

Reply:
Did this work before? Also I feel this should be configurable if we are to do this (probably a kwarg as its not something that will regularly be configured I guess)

Reply:
We can just remove it. It would not be a sensible way to use the functions, as we will not want the angle of intersection with the grid edge.